packages/postcss-minify-selectors/src/__tests__/index.js

```diff
@@ -96,10 +96,10 @@ test(
 );
 
 test(
-  'should sort using natural sort',
```

Comment:
I think natural sort has better compression... I think we should check it

Reply:
I am getting the opposite result, natural sort is slightly worse. I am checking with:
```
gzip -6 -c packages/cssnano-preset-default/src/__tests__/integrations/*css | wc -c
```
I get 32411 bytes for natural sort and 32407 bytes for the built-in sort.

Reply:
Gzipping all files to disk gives the same result for both: 356K